pkg/minikube/cluster/cluster_windows.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/docker/machine/drivers/hyperv"
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/golang/glog"
+	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows/registry"
+)
+
+func createHypervHost(config MachineConfig) drivers.Driver {
+	d := hyperv.NewDriver(constants.MachineName, constants.Minipath)
+	d.Boot2DockerURL = config.GetISOFileURI()
+	d.VSwitch = config.HypervVirtualSwitch
+	d.MemSize = config.Memory
+	d.CPU = config.CPUs
+	d.DiskSize = int(config.DiskSize)
+	d.SSHUser = "docker"
+	return d
+}
+
+func detectVBoxManageCmd() string {
+	cmd := "VBoxManage"
+	if p := os.Getenv("VBOX_INSTALL_PATH"); p != "" {
+		if path, err := exec.LookPath(filepath.Join(p, cmd)); err == nil {
+			return path
+		}
+	}
+
+	if p := os.Getenv("VBOX_MSI_INSTALL_PATH"); p != "" {
+		if path, err := exec.LookPath(filepath.Join(p, cmd)); err == nil {
+			return path
+		}
+	}
+
+	// Look in default installation path for VirtualBox version > 5
+	if path, err := exec.LookPath(filepath.Join("C:\\Program Files\\Oracle\\VirtualBox", cmd)); err == nil {
+		return path
+	}
+
+	// Look in windows registry
+	if p, err := findVBoxInstallDirInRegistry(); err == nil {
+		if path, err := exec.LookPath(filepath.Join(p, cmd)); err == nil {
+			return path
+		}
+	}
+
+	if path, err := exec.LookPath(cmd); err == nil {
+		return path
+	}
+	return cmd
+}
```

Comment:
Same as above. 
This something @anjannath added for #1822. Also we should remove detectVBoxManageCmd functions as these are not used anywhere.

Reply:
his implementation could exist for now... but should be deprecated over time to move to a general detection.